gitensite/apps/bookinfo/models.py

```diff
@@ -57,7 +57,10 @@ def subjects_str(self):
     
     @property
     def author_first_last(self):
-        return " ".join(self.author.name.split(", ")[::-1])
+        if self.author is None:
+            return None
+        else:
+            return " ".join(self.author.name.split(", ")[::-1])
```

Comment:
test this for unicode author names, for example #11299

Reply:
or even better #15008

Reply:
Tested with #15008 and it is working fine.
![image](https://user-images.githubusercontent.com/2292715/37879373-08968530-3046-11e8-805a-0b060e029248.png)